### PR TITLE
Add pt tokens

### PR DESCRIPTION
**Pull request by amishas157**
Ref: https://github.com/mapbox/geodata/pull/4091#issuecomment-732472015

### CODE REVIEW

File: tokens/pt.json [hunk 1]
```diff
@@ -808,10 +1101,12 @@
     {
         "tokens": [
             "CM",
-            "Câmara Municipal"
+            "Câmara Municipal",
```
Comment (ilissablech) [inline tokens/pt.json:1104]: We strip out accents and convert everything to lowercase so line 1104 & 1105 are duplicates

File: tokens/pt.json [hunk 2]
```diff
@@ -325,6 +556,52 @@
         "note": "translates to 'road'",
         "type": "way"
     },
+    {
+        "tokens": [
```
Comment (ilissablech) [inline tokens/pt.json:560]: any tokens that are multi-word need to be flagged with `"spanBoundaries": <number>` . [example](https://github.com/mapbox/geocoder-abbreviations/blob/master/tokens/es.json#L1638-L1644)

Comment (amishas157) [reply tokens/pt.json:560]: @ilissablech if I use 

```
    {
        "tokens": [
            "CM",
            "camara municipal",
            "c.m.",
            "c. m."
        ],
        "full": "camara municipal",
        "canonical": "CM",
        "spanBoundaries": 1
    }
```

test throws error:
```
not ok 80 all phrases except the canonical one must span the same number of boundaries: ["CM","camara municipal","c.m.","c. m."]
```

Should I be splitting it into two rules? :thinking:

Comment (ilissablech) [reply tokens/pt.json:560]: @amishas157 yup. we have an example in [French](https://github.com/mapbox/geocoder-abbreviations/blob/master/tokens/fr.json#L721-L742)

File: tokens/pt.json [hunk 3]
```diff
@@ -808,10 +1107,18 @@
...
+            "c. m."
         ],
-        "full": "Câmara Municipal",
+        "full": "camara municipal",
         "canonical": "CM",
         "spanBoundaries": 1
+    },
+    {
+        "tokens": [
+            "c.m."
```
Comment (ilissablech) [inline tokens/pt.json:1119]: @amishas157 periods get stripped out so `c.m.` == "CM" (listed in the token set above). this token can be removed.

Comment (amishas157) [reply tokens/pt.json]: Addressed  in https://github.com/mapbox/geocoder-abbreviations/pull/89/commits/4099d02c0bc1de896fc131d7ed8a45c111e68d49

**Review by ilissablech — APPROVED** [review]
LGTM

**Pull request merged**